test/e2e/app-dir/use-params/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
Bonus points if we default to `true` here in our tests. Test authors should opt-into disabling lib check. `"skipLibCheck": false,` really helps getting more test coverage for our types.